src/stream/src/task/stream_manager.rs

```diff
@@ -661,12 +658,7 @@ impl LocalStreamManagerCore {
                 self.streaming_metrics.clone(),
                 self.config.unique_user_stream_errors,
             );
-            let vnode_bitmap = actor
-                .vnode_bitmap
-                .as_ref()
-                .map(|b| b.try_into())
```

Comment:
Ref: BitMap's `TryFrom` changed to `From` in https://github.com/risingwavelabs/risingwave/pull/4364